src/PowerShellEditorServices/Analysis/AnalysisService.cs

```diff
@@ -24,7 +24,7 @@ public class AnalysisService : IDisposable
     {
         #region Private Fields
 
-        private const int NumRunspaces = 2;
+        private const int NumRunspaces = 1;
```

Comment:
Runspacepool does the queuing of commands for us. If we do not use runspacepool then we will need to back to using a lock to prevent concurrent PS invocation.